### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Draniu**
Added NetBeans plugin 

https://github.com/junichi11/cakephp3-netbeans

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -363,6 +363,7 @@ Additional lists you might find useful:
 - [Cakebox](https://github.com/alt3/cakebox) - A Vagrant development environment powered by the CakePHP 3.x Console.
 - [CakePHP.gitignore](https://github.com/github/gitignore/blob/master/CakePHP.gitignore) - The .gitignore file proposals.
 - [PhpStorm plugin](https://github.com/skie/PhpStorm) -  CakePHP auto-completion support for console commands in PhpStorm IDE.
+- [NetBeans plugin](https://github.com/junichi11/cakephp3-netbeans#netbeans-cakephp3-plugin-dev) -  This plugin provides support for CakePHP3 in NetBeans 8.1+.
```
Comment (ADmad) [inline README.md]: Please follow alphabetical order and remove hash from url.

Comment (dereuromark) [inline README.md]: Also the 3 is not necessary. Everything here is for 3.x

Comment (Draniu) [inline README.md]: I've ad  3 because cake plugin is in netbeans repository is for cake 2

Comment (dereuromark) [inline README.md]: I meant in the description

Comment (dereuromark) [inline README.md]: But in general it is also wiser to keep versions out of the URL an Name in general. To be future proof.

Comment (Draniu) [inline README.md]: done

**Pull request merged**